lib/matplotlib/axes/_axes.py

```diff
@@ -5049,11 +5095,29 @@ def reduce_C_function(C: array) -> float
                 ymin = 10.0 ** ymin
                 ymax = 10.0 ** ymax
                 self.set_yscale(yscale)
+
+            if xscale == 'symlog':
+                polygons[:, :, 0] = inv_symlog_trans['x'].transform_non_affine(
```

Comment:
```suggestion
                polygons[:, :, 0] = inv_symlog_trafo_x.transform_non_affine(
```
And likewise for y below.